lib/pace/Install.py

```diff
@@ -55,55 +57,68 @@
                     help="set version of PACE library to download and build (default: %s)" % version)
 parser.add_argument("-vv", "--verbose", action="store_true",
                     help="be more verbose about is happening while this script runs")
+parser.add_argument("-l", "--local", default=None,
```

Comment:
Please follow the conventions established in the other Install.py files for referencing local installations:
- use the -p flag to reference the local installation folder
- create/use `liblink` and `includelink` symbolic links
and so on.
Please see hdnnp, latte, mscg, plumed, voronoi folders, for example.